src/local/butler/scripts/migration/monorail_to_issuetracker_migrator.py

```diff
@@ -25,7 +25,8 @@ def execute(args):
   and/or group_bug_information fields to reflect the Issue Tracker issue
   id rather than the Monorail issue id."""
 
-  issue_id_dict = get_monorail_issuetracker_issue_id_dictionary(args.file_loc)
+  issue_id_dict = get_monorail_issuetracker_issue_id_dictionary(
+      args.file_loc, args.roll_back)
 
   testcases = []
 
```

Comment:
As @oliverchang noted yesterday in #3648, you don't need to wrap the values assigned to fields in `ndb.{String,Integer}Property()`. In fact, it might break things - those functions are used to *define* fields in `ndb.Model` classes. You can just do:

```
testcase.bug_information = issue_id_dict[testcase.bug_information]
```
And similarly for `group_bug_information`.

Reply:
Done.